packages/@aws-cdk/aws-lambda-event-sources/test/integ.dynamodb.ts

```diff
@@ -20,7 +20,8 @@ class DynamoEventSourceTest extends cdk.Stack {
 
     fn.addEventSource(new DynamoEventSource(queue, {
       batchSize: 5,
-      startingPosition: lambda.StartingPosition.TRIM_HORIZON
+      startingPosition: lambda.StartingPosition.TRIM_HORIZON,
+      enabled: false
```

Comment:
Avoid changing an existing integration test. 

Actually, an integration test is not needed for this feature; unit tests are sufficient.